all-about-that-bash.sh

```diff
@@ -879,4 +879,13 @@ try-land() {
     git fetch --no-tags origin develop && git rebase origin/develop && arc land --onto develop
 }
 
-trap 'echo -e "${clPURPLE}-- Started at $(date +"%H:%M:%S") --${cLIGHTGRAY}"' DEBUG
\ No newline at end of file
+#pre-requisite: checked out on develop branch
+git-land() {
+    git fetch --no-tags origin develop; git rebase; git push;
+}
+
+git-land-multi() {
+    git-land; git-land; git-land; git-land; git-land;
```

Comment:
why not use param n to put the git-land into for-loop?

Reply:
how to do that :sweat_smile:

Reply:
forgive this fool apprentice, shifu :bowing_man:

Reply:
```
loop-for() {
	for (( c=1; c<=${1}; c++ ))
	do  
	   echo "this is $c"
	done
}
```
${1} is the first param.
when executing 
```
loop-for 2
```
where ${1} = 2, the result will be:
```
this is 1
this is 2
```

Reply:
if it is just `loop-for`, what it will be look like?

Reply:
TL;DR I want to make just a single command.
 If you put number n, it will do n times, if you don't, it will do 1 time.